pkg/kubectl/stop.go

```diff
@@ -337,6 +333,8 @@ func (reaper *StatefulSetReaper) Stop(namespace, name string, timeout time.Durat
 	}
 	if timeout == 0 {
 		numReplicas := ss.Spec.Replicas
+
+		// BUG: this timeout is never used.
```

Comment:
> I wonder if line 340 below should be using timeout rather than reaper.timeout (by looking how it's done throughout the file)

Maybe

I found there maybe another bug in https://github.com/alexandercampbell/kubernetes/blob/d3554ac0ce87f75e6976e97c61cc81165207aeb8/pkg/kubectl/stop.go#L294-L329. Argument `timeout` is never used in this function.